### PR TITLE
Rename linked collections to projects

### PR DESCRIPTION
**Pull request by nickvergessen**
@juliushaertl I guess the remaining occurrences are from your library?

### CODE REVIEW

**Comment by juliushaertl** [comment]
@nickvergessen The rest should be fixed if you bump nextcloud-vue-collections to 0.4.0

File: js/app.js [hunk 1]
```diff
@@ -301,7 +301,7 @@
 					this.setRoom(model);
 				}
 			});
-			this._sidebarView.addTab('collections', { label: t('spreed', 'Collections'), icon: 'icon-category-integration' }, this._collectionsView);
+			this._sidebarView.addTab('collections', { label: t('spreed', 'Projects'), icon: 'icon-category-integration' }, this._collectionsView);
```
Comment (juliushaertl) [inline js/app.js]: Icon class should be icon-projects once https://github.com/nextcloud/server/pull/14990 is in.

Comment (nickvergessen) [reply js/app.js]: Adjusted

**Comment by nickvergessen** [comment]
Updated, thanks for the lib update

**Pull request merged**